src/ecalc/libraries/libecalc/common/libecalc/core/consumers/consumer_system.py

```diff
@@ -109,29 +109,43 @@ def evaluate(
         variables_map: VariablesMap,
         temporal_operational_settings: TemporalModel[List[SystemOperationalSettings]],
     ) -> EcalcModelResult:
+        """
+        Evaluating a consumer system that may be composed of both consumers and other consumer systems. It will default
+        to the last operational setting if all settings fails.
+
+        Notes:
+            - We use 1-indexed operational settings output. Should consider to change this to something else.
+
+        Fixme: Regularity and units are correct by chance. We need to carry this information explicitly.
+        Todo: Need to validate energy usage units (stream day vs. calendar day).
+        Todo: We are missing model results for each operational setting tested....
+            Since we will abandon model results, we could just output all single operational settings results
+            into a big ball og mud in the models result?
+        """
         is_valid = TimeSeriesBoolean(
             timesteps=variables_map.time_vector, values=[False] * len(variables_map.time_vector), unit=Unit.NONE
         )
         energy_usage = TimeSeriesRate(
             timesteps=variables_map.time_vector,
             values=[0] * len(variables_map.time_vector),  # TODO: Initialize energy usage to NaN not zero 4084
-            unit=Unit.STANDARD_CUBIC_METER_PER_DAY,
+            unit=Unit.NONE,
             regularity=temporal_operational_settings.models[0].model[0].rates[0].regularity,
         )
         power = TimeSeriesRate(
             timesteps=variables_map.time_vector,
-            values=[math.nan] * len(variables_map.time_vector),
+            values=[0] * len(variables_map.time_vector),
```

Comment:
why change this? `math.nan` should indicate that it doesnt apply. `0` indicates that it evaluates to 0...

Reply:
First I want to make this behave the same as the old consumer system :)